icicle/appUtils/ntt/tests/verification.cu

```diff
@@ -19,29 +19,65 @@ typedef curve_config::scalar_t test_data;
 void random_samples(test_data* res, uint32_t count)
 {
   for (int i = 0; i < count; i++)
+    // res[i] = i==0 ? test_scalar::one() : test_scalar::zero();
     res[i] = i < 1000 ? test_data::rand_host() : res[i - 1000];
 }
 
 void incremental_values(test_scalar* res, uint32_t count)
 {
   for (int i = 0; i < count; i++) {
+    // res[i] = i%64 ? res[i - 1] + test_scalar::one() : test_scalar::zero();
     res[i] = i ? res[i - 1] + test_scalar::one() : test_scalar::zero();
   }
 }
 
+__global__ void copy_batch(test_scalar* in, test_scalar* out, int row_size, int column_size){
+  int tid = blockDim.x * blockIdx.x + threadIdx.x;
+  if (tid >= row_size * column_size) return;
+  out[tid] = in[tid];
+}
+
+__global__ void copy_batch2(test_scalar* in, test_scalar* out, int row_size, int column_size){
```

Comment:
why need 3 such functions? same for transpose. If you need them, please give clear names

Reply:
removed